diff --git a/Makefile b/Makefile
index 036691c..645eee8 100644
--- a/Makefile
+++ b/Makefile
@@ -6,9 +6,11 @@ proj-version := $(proj-major).$(proj-minor)
 CFLAGS := -Wall -g
 LDFLAGS :=
 
-prefix := /usr/local
+multiarch = $(shell dpkg-architecture -q DEB_TARGET_MULTIARCH)
+
+prefix := /usr
 bindir := $(prefix)/bin
-libdir := $(prefix)/lib
+libdir := $(prefix)/lib/$(multiarch)
 includedir := $(prefix)/include
 servicedir := $(prefix)/lib/systemd/system
 
